os/drivers/lwnl/lwnl_evt_queue.c

```diff
@@ -181,15 +181,20 @@ int lwnl_add_event(lwnl_cb_status type, void *buffer, uint32_t buf_len)
 	evt->data.data = NULL;
 	evt->data.data_len = 0;
 	if (buffer) {
-		char *output = kmm_malloc(buf_len);
-		if (!output) {
-			LWNL_LOGE(TAG, "fail to alloc buffer");
-			kmm_free(evt);
-			return -3;
+		if (buf_len < 0) {
```

Comment:
Could you leave comment about the case that `buf_len` is smaller than 0?

Reply:
As I commented in this commit,  if data_len is negative, a queue copies pointer of data.

```suggestion
	trble_post_event(LWNL_EVT_BLE_CLIENT_NOTI, data, -(size));
```